### PR TITLE
conversion of string and dimen to resource form

### PR DESCRIPTION
**Pull request by adithya61**
commit squashed
and .idea file changes not included.

### CODE REVIEW

File: Scorecounter/app/src/main/res/values/colors.xml [hunk 1]
```diff
@@ -1,6 +1,2 @@
 <?xml version="1.0" encoding="utf-8"?>
-<resources>
-    <color name="colorPrimary">#87cefa</color>
-    <color name="colorPrimaryDark">#000000</color>
-    <color name="colorAccent">#D81B60</color>
-</resources>
```
Comment (chanmol1999) [inline Scorecounter/app/src/main/res/values/colors.xml:6]: Reason to remove this?

File: Scorecounter/app/src/main/res/values/styles.xml [hunk 2]
```diff
@@ -1,11 +1,5 @@
 <resources>
 
     <!-- Base application theme. -->
-    <style name="AppTheme" parent="Theme.AppCompat.Light.DarkActionBar">
-        <!-- Customize your theme here. -->
-        <item name="colorPrimary">@color/colorPrimary</item>
-        <item name="colorPrimaryDark">@color/colorPrimaryDark</item>
-        <item name="colorAccent">@color/colorAccent</item>
-    </style>
```
Comment (chanmol1999) [inline Scorecounter/app/src/main/res/values/styles.xml]: Same why have you removed the styles?

**Review by chanmol1999 — CHANGES_REQUESTED** [review]

File: Scorecounter/app/src/main/res/values/colors.xml (hunk 1 above)
Comment (adithya61) [reply Scorecounter/app/src/main/res/values/colors.xml:6]: I don't remember removing this code.

File: Scorecounter/app/src/main/res/values/styles.xml (hunk 2 above)
Comment (adithya61) [reply Scorecounter/app/src/main/res/values/styles.xml]: Even this one

**Comment by adithya61** [comment]
changes made

**Review by chanmol1999 — APPROVED** [review]

**Pull request merged**